packages/cli/src/telemetry/index.ts

```diff
@@ -53,33 +82,41 @@ export class Telemetry {
 			return Promise.resolve();
 		}
 
-		const allPromises = Object.keys(this.executionCountsBuffer).map(async (workflowId) => {
+		const allPromises = Object.keys(this.executionCountsBuffer.counts).map(async (workflowId) => {
 			const promise = this.track('Workflow execution count', {
+				version_cli: this.versionCli,
 				workflow_id: workflowId,
-				...this.executionCountsBuffer[workflowId],
+				...this.executionCountsBuffer.counts[workflowId],
+				...this.executionCountsBuffer.firstExecutions,
 			});
-			this.executionCountsBuffer[workflowId].manual_error_count = 0;
-			this.executionCountsBuffer[workflowId].manual_success_count = 0;
-			this.executionCountsBuffer[workflowId].prod_error_count = 0;
-			this.executionCountsBuffer[workflowId].prod_success_count = 0;
+
+			this.executionCountsBuffer.counts[workflowId].manual_error_count = 0;
+			this.executionCountsBuffer.counts[workflowId].manual_success_count = 0;
+			this.executionCountsBuffer.counts[workflowId].prod_error_count = 0;
+			this.executionCountsBuffer.counts[workflowId].prod_success_count = 0;
 
 			return promise;
 		});
 
-		allPromises.push(this.track('pulse'));
+		allPromises.push(this.track('pulse', { version_cli: this.versionCli }));
 		return Promise.all(allPromises);
 	}
 
 	async trackWorkflowExecution(properties: IDataObject): Promise<void> {
 		if (this.client) {
 			const workflowId = properties.workflow_id as string;
-			this.executionCountsBuffer[workflowId] = this.executionCountsBuffer[workflowId] ?? {
+			this.executionCountsBuffer.counts[workflowId] = this.executionCountsBuffer.counts[
```

Comment:
would it be better for backward compatibility to simply add the first execution keys to counts?

Reply:
counts are per wfs, we only  need the first execution by kind